From 9fb7520f46a0cc1e13d4a7fcbcc5d9c04afbf27f Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Thu, 27 Apr 2017 15:32:15 +0300
Subject: [PATCH 0986/1345] net: mvpp2x: move FIFO definitions to same header
 file

commit  e7640fc17bfde76af3057bf4a6102dca1194863d from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Move all FIFO definitions to mv_pp2x_hw_type.h
No functionality changes.

Change-Id: I2fa5c8b71c11e2ac7ec82decad8ebc483efb9b78
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/38988
Reviewed-by: Omri Itach <omrii@marvell.com>
Reviewed-by: Yuval Caduri <cyuval@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/ethernet/marvell/mvpp2x/mv_pp2x.h      |   10 ----------
 .../net/ethernet/marvell/mvpp2x/mv_pp2x_hw_type.h  |   10 ++++++++++
 2 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x.h b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x.h
index 7481bcd..a083f516 100644
--- a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x.h
+++ b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x.h
@@ -162,16 +162,6 @@
 #define QV_THR_2_CPU(sw_thread_id)	(sw_thread_id - first_addr_space)
 #define QV_CPU_2_THR(cpu_id)		(first_addr_space + cpu_id)
 
-/* TX FIFO constants */
-#define MVPP2_TX_FIFO_DATA_SIZE_10KB		0xa
-#define MVPP2_TX_FIFO_DATA_SIZE_3KB		0x3
-
-#define MVPP2_TX_FIFO_MINIMUM_THRESHOLD		256
-#define MVPP2_TX_FIFO_THRESHOLD_10KB	(MVPP2_TX_FIFO_DATA_SIZE_10KB * 1024 - \
-					MVPP2_TX_FIFO_MINIMUM_THRESHOLD)
-#define MVPP2_TX_FIFO_THRESHOLD_3KB	(MVPP2_TX_FIFO_DATA_SIZE_3KB * 1024 - \
-					MVPP2_TX_FIFO_MINIMUM_THRESHOLD)
-
 /* Used for define type of data saved in shadow: SKB or extended buffer or nothing */
 #define MVPP2_ETH_SHADOW_SKB		0x1
 #define MVPP2_ETH_SHADOW_EXT		0x2
diff --git a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_hw_type.h b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_hw_type.h
index 0122895..8e4d1fe 100644
--- a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_hw_type.h
+++ b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_hw_type.h
@@ -1227,6 +1227,16 @@
 /* Descriptor alignment mask */
 #define MVPP2_TX_DESC_ALIGN		(MVPP2_DESC_ALIGNED_SIZE - 1)
 
+/* TX FIFO constants */
+#define MVPP2_TX_FIFO_DATA_SIZE_10KB		0xa
+#define MVPP2_TX_FIFO_DATA_SIZE_3KB		0x3
+
+#define MVPP2_TX_FIFO_MINIMUM_THRESHOLD		256
+#define MVPP2_TX_FIFO_THRESHOLD_10KB	(MVPP2_TX_FIFO_DATA_SIZE_10KB * 1024 - \
+					MVPP2_TX_FIFO_MINIMUM_THRESHOLD)
+#define MVPP2_TX_FIFO_THRESHOLD_3KB	(MVPP2_TX_FIFO_DATA_SIZE_3KB * 1024 - \
+					MVPP2_TX_FIFO_MINIMUM_THRESHOLD)
+
 /* RX FIFO constants */
 #define MVPP2_RX_FIFO_PORT_DATA_SIZE	0x2000
 #define MVPP2_RX_FIFO_PORT_ATTR_SIZE	0x80
-- 
1.7.9.5

